tests/test_file.py

```diff
@@ -14,7 +14,13 @@
 
 import pytest
 import os
-import pathlib
+try:
+    import pathlib
+except ImportError as e:
+    try:
```

Comment:
Ditto, and, I guess you get the point. The following do not need the try either.